collections/_article/godot-openxr-vendors-plugin-400.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Godot OpenXR Vendors Plugin 4.0.0"
+excerpt: "What's new in the latest release of the Godot OpenXR Vendors plugin?"
+categories: ["progress-report"]
+author: "David Snopek"
+image: /storage/blog/covers/xxx
+image_caption_description: "Photo by xxx"
+date: 2025-07-23 12:00:00
+---
+
+# Godot OpenXR Vendors Plugin 4.0.0
+
+OpenXR support has been built into Godot, since the release of Godot 4.0. However, the [OpenXR Vendors plugin](https://github.com/GodotVR/godot_openxr_vendors) (maintained by Godot's XR team) includes extensions to OpenXR created by hardware vendors (ex Meta, Pico, HTC, etc), which we've chosen to keep outside of Godot itself.
```

Comment:
```suggestion
OpenXR support has been built into Godot since the release of Godot 4.0. However, the [OpenXR Vendors plugin](https://github.com/GodotVR/godot_openxr_vendors) (maintained by Godot's XR team) includes extensions to OpenXR created by hardware vendors (e.g. Meta, Pico, HTC, etc.), which we've chosen to keep outside of Godot itself.
```